### ISSUE TITLE
[BITV] 9.1.1.1a/3.4 - Mobile (responsive) view - The button to close/open the navigation bar on the left side does not have an alternative text. (6)

### ISSUE THREAD

**Issue by Pytal** [nextcloud/server#36717]
- [x] Fix

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v3-n4

![image](https://user-images.githubusercontent.com/24800714/218900233-aec73306-a301-447a-a5fe-3794df722826.png)

</details>

- [x] 9.1.1.1a/5.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v5-n2
- [x] 9.1.1.1a/6.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v6-n1
- [x] 9.1.1.1a/7.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v7-n1
- [x] 9.1.1.1a/9.1 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v9-n1
- [x] 9.1.1.1a/10.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-cd239865a5-v10-n4

**Comment by Pytal** [comment]
Label present on `master` from NcAppNavigation
